austation/code/modules/coilgun/charger.dm

```diff
@@ -0,0 +1,152 @@
+// The base used for calculating projectile speed increase
+// lower values make speed increases more diminishing
+#define BASE 0.9975
+
+// the smallest amount of power the charger can use to function in watts.
+#define POWER_DIVIDER 100000
```

Comment:
and also apparently the factor used to determine a multiplier for the projectile speed